tests/kernel/mutex/mutex_api/src/test_mutex_apis.c

```diff
@@ -386,21 +387,16 @@ void test_mutex_priority_inheritance(void)
 	k_msleep(TIMEOUT+1000);
 }
 
-/*test case main entry*/
-void test_main(void)
+static void *mutex_api_tests_setup(void)
 {
+#ifdef CONFIG_USERSPACE
```

Comment:
I think we can remove the #ifdef  because when CONFIG_USERSPACE=n, k_thread_access_grant will do nothing.

Reply:
Maybe this can make things a bit explicit.